apps/web/src/config/index.ts

```diff
@@ -34,13 +34,10 @@ export const SENTRY_DSN = window._env_.REACT_APP_SENTRY_DSN || process.env.REACT
 
 export const ENV = window._env_.REACT_APP_ENVIRONMENT || process.env.REACT_APP_ENVIRONMENT;
 
-const blueprintApiUrlByEnv =
-  ENV === 'production' || ENV === 'prod' ? 'https://api.novu.co' : 'https://api.novu-staging.co';
-
```

Comment:
This was set to these URLs also for EU customers, which resulted in their EU Clerk instance token being invalid against our US API.

On API error, we clear the env-id request header from local storage (not 100% sure why) resulting in either fallback to dev environment or no environment being send in following requests.